packages/create-stylable-app/template/ts-react-rollup/template.js

```diff
@@ -1,7 +1,8 @@
 /** @type {import('create-stylable-app').TemplateDefinition} */
 module.exports = {
-    dependencies: ['@stylable/runtime', 'react', 'react-dom'],
```

Comment:
why remove this?

Reply:
Why did we need it in the first place (not as dev dep)? 
It was used for types as far as it looks.